.github/workflows/sync-strategies.yml

```diff
@@ -0,0 +1,26 @@
+name: Sync ClusterBuildStrategies
+on:
+  push:
```

Comment:
I think it is better to keep this pipeline manually as for release pipeline 
here is an example https://github.com/shipwright-io/build/blob/main/.github/workflows/release.yaml
So we can merge things there, test them, and once we need to release the release-strategies.yaml, we hit the pipeline